.github/workflows/version-code-and-release.yml

```diff
@@ -73,6 +73,19 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Set github config
+        run: |
+          git config --global user.email "72879786+protectionsmachine@users.noreply.github.com"
+          git config --global user.name "protectionsmachine"
+
+      - name: Extract version from pyproject.toml and create tag
+        id: extract_version
+        run: |
+          version=$(grep '^version = ' pyproject.toml | cut -d '"' -f2)
+          echo "Detected version: $version"
+          git tag -a "v$version" -m "Release version $version"
+          git push origin "v$version"
+
       - name: Run Release Drafter
```

Comment:
@traut was going to gate this behind only major/minor tags, but since its only generating a draft, I now dont think we need to.